conda.recipe/run_test.py

```diff
@@ -0,0 +1 @@
+import datashader; datashader.test()
```

Comment:
This means the package can be tested on the install side without forcing the user to install from source.